webfonts/Roboto/Fonts.css

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
I'm not sure if we need that file anymore, or at least we may not need the last part that works around the font loading issue.

Reply:
ah, ok I'll see what happens on a WASM app if I don't include this Fonts.css file. I took this from the [Extensions templates](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Wasm/WasmCSS/Fonts.css) so we'll need to adjust that accordingly as well I guess